reference_system_autoware/include/reference_system_autoware/node/command.hpp

```diff
@@ -53,11 +53,11 @@ class Command : public rclcpp::Node
     const int64_t accumulated_latency_in_ns =
       input_accumulated_latency + timestamp_in_ns - input_timestamp;
 
-    std::cout << "\nreceived message stats:\n";
-    std::cout << "  current timestamp in ns: " << timestamp_in_ns << std::endl;
-    std::cout << "  message timestamp in ns: " << input_timestamp << std::endl;
-    std::cout << "  accumulated latency in ns: " << accumulated_latency_in_ns <<
-      std::endl;
```

Comment:
@flynneva 

One minor thing. If you would like to comment out large  code blocks I prefer the macro style:
```cpp
#if 0
  code block
#endif
```

The reason is that when you comment/uncomment this code block in a PR the changes are only two lines and are independent of the amount of code one commented.